R/extract.R

```diff
@@ -9050,3 +9050,96 @@ extract.hurdle <- extract.zeroinfl
 #' @export
 setMethod("extract", signature = className("hurdle", "pscl"),
           definition = extract.hurdle)
+
+# -- extract.logitr (logitr >= 0.8.0) ------------------------------------------
+
+#' @noRd
+extract.logitr <- function(model,
+                           include.nobs   = TRUE,
```

Comment:
Remove the spaces before the `=` to make the formatting compatible with the remaining code.